sdks/python/apache_beam/examples/complete/juliaset/setup.py

```diff
@@ -30,7 +30,8 @@
 import subprocess
 
 import setuptools
-from setuptools.command.build import build as _build  # type: ignore
+
+from distutils.command.build import build as _build  # isort:skip
```

Comment:
Add comments why this is has to be skipped so that somebody doesn't try to clean it up.

Reply:
Done